src/app/pages/swap/components/selected-asset-field.tsx

```diff
@@ -1,46 +1,44 @@
 import { Field } from 'formik';
-import { Box, Flex, HStack, styled } from 'leather-styles/jsx';
-
-import { Flag } from '@app/components/layout/flag';
+import { Box, Flex, HStack } from 'leather-styles/jsx';
 
 interface SelectedAssetFieldProps {
   contentLeft: React.JSX.Element;
   contentRight: React.JSX.Element;
-  icon?: string;
   name: string;
+  showError?: boolean;
 }
 export function SelectedAssetField({
   contentLeft,
   contentRight,
-  icon,
   name,
+  showError,
 }: SelectedAssetFieldProps) {
   return (
     <Flex
       alignItems="center"
       border="1px solid"
-      borderColor="accent.border-default !important"
+      borderColor={showError ? 'error !important' : 'accent.border-default !important'}
       borderRadius="10px"
+      css={{
+        '&:focus-within': {
+          border: '1px solid',
```

Comment:
There already is a 1px border set. Don't we want to change its colour also when not an error so it's clear the input is focused?

Reply:
Also why css and not the prop `_focusWithin` ?

Reply:
https://panda-css.com/docs/guides/dynamic-styling

Reply:
I tried `_focusWithin` here but didn't work, but can try again. Also, this seems to be a design question bc it didn't have a focus state.

Reply:
I think it's not a `_focusWithin` problem, but that you're trying to apply dynamic styles in a way not supported by panda

Reply:
Yeah, so, I am failing to understand your question then abt css and not `_focusWithin`? Why is using the `css` prop bad here if working?